index.html

```diff
@@ -80,6 +80,7 @@ <h1>Hey there. I’m Faith<br />
             </div>
         </div>
     </section>
+
     <section class='recent-works'>
         <div class="wrapper">
             <div class="dropdown"></div>
```

Comment:
Please note that in the project requirements all the recent works section will be displayed from the projectDatas you have, not only the modals (Popup) in your project, In this index.html you have a card for every card from line 83 to 220, instead of creating it dynamically. You did great with modals so it will be easy to do it with cards.

- [ ] Please load the recent works section dynamically for each project, in that way adding projects in the future will be easy.

Reply:
Thanks for the review @Felipeg005 

I have implemented the changes 👍